resources/assets/less/bem/beatmap-scoreboard-table.less

```diff
@@ -18,41 +18,62 @@
 
 .beatmap-scoreboard-table {
   @_top: beatmap-scoreboard-table;
+  @activated-row: beatmap-scoreboard-table__body-row--menu-active;
+  @friend-row: beatmap-scoreboard-table__body-row--friend;
+  @passive-row: beatmap-scoreboard-table__body-row--highlightable;
+  @self-row: beatmap-scoreboard-table__body-row--self;
+
+  @handle: 16px;
   @row-height: 1.8em;
 
-  width: 100%;
-  overflow-x: auto;
+  .cell-color(@color, @highlight) {
+    color: @color;
 
-  .row-colour(@colour) {
-    > td {
-      background-color: lighten(@colour, 20%);
-    }
+    && {
+      .@{activated-row} & {
+        color: @highlight;
+      }
+
+      .@{passive-row}:hover & {
+        color: @highlight;
 
-    &:nth-child(odd) {
-      > td {
-        background-color: lighten(@colour, 20%);
+        .@{_top}--menu-active & {
+          color: @color;
+        }
       }
     }
+  }
 
-    &:hover {
-      > td {
-        background-color: @colour;
+  .row-color(@name, @color) {
+    .@{activated-row}.@{name} & {
+      color: #fff;
+      background-color: @color;
+    }
+
+    .@{passive-row}.@{name} & {
+      background-color: lighten(@color, 20%);
+    }
+
+    .@{passive-row}.@{name}:hover & {
+      color: #fff;
+      background-color: @color;
+
+      .@{_top}--menu-active & {
+        color: unset;
```

Comment:
this turns gray `0` to black. Also weird highlight is disabled but user card is still shown. The invisible menu burger button also still works. A `pointer-events: none` at top level might be simpler?